Neos.Flow/Classes/Http/Cookie.php

```diff
@@ -142,8 +142,10 @@ public function __construct($name, $value = null, $expires = 0, $maximumAge = nu
         if ($path !== null && preg_match(self::PATTERN_PATH, $path) !== 1) {
             throw new \InvalidArgumentException('The parameter "path" passed to the Cookie constructor must be a valid path as per RFC 6265, Section 4.1.1.', 1345123078);
         }
-        if (!\in_array($sameSite, [self::SAMESITE_LAX, self::SAMESITE_STRICT, self::SAMESITE_NONE, null], true)) {
-            throw new \InvalidArgumentException('The parameter "sameSite" passed to the Cookie constructor must be a valid samesite value. Possible values are "none", "strict", "lax" and null', 1584955500);
+
+        $sameSite = $sameSite === null ? self::SAMESITE_LAX : $sameSite;
```

Comment:
Sorry, but one last nitpick:

```suggestion
        if ($sameSite === null) {
            $sameSite = self:: SAMESITE_LAX;
        }
```
or
```suggestion
        $sameSite = $sameSite ?? self::SAMESITE_LAX;
```
is much easier to comprehend IMO